src/main/resources/agencyFinder/prison-codes.csv

```diff
@@ -14,26 +14,36 @@ GTI,HMP Gartree
 GNI,HMP Grendon
 HEI,HMP Hewell
 HOI,HMP High Down
+HPI,HMP Highpoint
 HLI,HMP Hull
 LEI,HMP Leeds
 LCI,HMP Leicester
 LWI,HMP Lewes
 LII,HMP Lincoln
+LHI,HMP Lindholme
 LPI,HMP Liverpool
 MSI,HMP Maidstone
 MRI,HMP Manchester
+NMI,HMP Nottingham
 PVI,HMP Pentonville
 PFI,HMP Peterborough
+RCI,HMP Rochester
+RHI,HMP Rye Hill
 SLI,HMP Swaleside
 WDI,HMP Wakefield
 WWI,HMP Wandsworth
-XXX,HMP Windmill Hill
```

Comment:
not sure why this was mapped to `XXX`, doesn't seem right to me

Reply:
No - I think that's the Gibraltar prison and I must have let this slip through.